src/Session.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Slides\Saml2;
+
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Facades\Cookie;
+use Slides\Saml2\Models\Tenant;
+use Slides\Saml2\Facades\Auth;
+use Slides\Saml2\Repositories\TenantRepository;
+use Slides\Saml2\OneLoginBuilder;
+use OneLogin\Saml2\Error as OneLoginError;
+
+/**
+ * Class Session
+ *
+ * @package Slides\Saml2
+ */
+class SamlSession
```

Comment:
Needs to be renamed to `Session`

Reply:
@natenatters Good catch! Fixed that now. Would you consider adopting this PR now that it is resolved?